templates/page.tmpl

```diff
@@ -124,7 +123,7 @@
 <img alt="" border="0" src="https://www.paypal.com/en_US/i/scr/pixel.gif" width="1" height="1">
 </form>
 <a class="FlattrButton" style="display:none;" rev="flattr;button:compact;"
-href="http://awesome.naquadah.org"></a>
+href="http://awesomewm.org"></a>
```

Comment:
Since @blueyed is the only one with expenses (is he?), I'm fine with him putting in whatever donation-thing he wants. If there are enough donations to cover the expenses for the domain, feel free to point that out. If you get rich, feel free to share. ;-)